scripts/seq_to_mts.py

```diff
@@ -78,6 +106,13 @@ def main(args):
         "median_l2fc": "LFC"
     }
 
+    # Get the columns for profile_id from SIG_COLS
```

Comment:
very minor, but maybe clarify in comment with
"Get the columns for the profile_id used in analysis pipeline from SIG_COLS"
just to be less confusion?